libraries/botbuilder-ai/src/qnamaker-interfaces/rankerTypes.ts

```diff
@@ -9,19 +9,19 @@
 /**
  * Enumeration of types of ranking.
  */
-export class RankerTypes {
+export enum RankerTypes {
```

Comment:
I _think_ this may be okay, @stevengum do you have thoughts? I'm trying to imagine a code scenario where this would break compilation and I'm drawing a blank.

Reply:
Changing `RankerTypes` to enum is to avoid type casting in QnAMakerDialog and also to fit the type `EnumExpression<RankerType>`.

Reply:
I'm drawing a blank as well... Nor can I think of any 100-level situations where this is being used. I think the delta is okay 👍🏼